### PR TITLE
REG-84 new regulome scoring algorithm

### PR DESCRIPTION
**Pull request by yunhailuo**

### CODE REVIEW

**Comment by yunhailuo** [comment]
Tests fail only `test_admin_user` as expected.

File: src/encoded/region_search.py [hunk 1]
```diff
@@ -402,20 +402,27 @@ def regulome_summary(context, request):
...
+        regulome_score = 'N/A'
         if rsids == [] and (int(end) - int(start)) > 1:
-            regulome_score = 'N/A'
             result['notifications'].append({coord: 'Failed: {}'.format(
                 'Non-SNP or multi-nucleotide region is not scorable')})
         else:  # Scorable
             try:
                 all_hits = region_get_hits(atlas, assembly, chrom, start, end)
-                regulome_score = atlas.regulome_score(all_hits['datasets'])
+                evidence = atlas.regulome_evidence(all_hits['datasets'])
```
Comment (hitz) [inline src/encoded/region_search.py]: I feel like regulome_summary and regulome_search have more than a little redundant code.   Discussed with @emmao making the summary do SNP fetches in the background asynchronous which might make most of this view code irrelvant.

**Comment by hitz** [comment]
Still need to test against the batch of SNPs with scores

**Comment by yunhailuo** [comment]
Shengcheng's batch of tests pass.
I tagged ENCD-4581 with [REVERT].
Will merge now.

**Pull request merged**